docker/Makefile

```diff
@@ -180,6 +196,41 @@ ub-18.04-irods-4.3.0.$(TAG): irods/ubuntu/18.04/Dockerfile ub-18.04-base.$(TAG)
 	--tag $(DOCKER_PREFIX)/ub-18.04-irods-4.3.0:$(TAG) --file $< ./irods
 	touch $@
 
+ub-22.04-irods-4.3-nightly.$(TAG): irods/ubuntu/22.04/Dockerfile ub-22.04-base.$(TAG)
```

Comment:
If at some point we are going to have a "stable" vs "nightly" of ub-22 + irods 4.3, it may be good to have this in a different path from `irods`. As the container is tagged as `ub-22.04-irods-4.3-nightly` I don't see a problem merging things as they are.